engine/execution/computation/computer/computer.go

```diff
@@ -196,12 +204,25 @@ func (e *blockComputer) executeBlock(
 		if err != nil {
 			return nil, fmt.Errorf("failed to execute collection at txIndex %v: %w", txIndex, err)
 		}
+		collectionID := collection.Guarantee.CollectionID[:]
+		blockLogger.Debug().
+			Hex("collection_id", collectionID).
+			Msg("collection executed")
 		bc.Commit(colView)
+		blockLogger.Debug().
+			Hex("collection_id", collectionID).
+			Msg("collection committed")
 		eh.Hash(res.Events[i])
+		blockLogger.Debug().
+			Hex("collection_id", collectionID).
+			Msg("collection events hashed")
```

Comment:
these maybe a bit too much, given that both Commit and Hash are async.

Reply:
I think the more accurate log line would be something like "collection events hashing initiated". In the callback, we can say "collection events hashed"